calcom/cal.com
# Feature/Routing-Forms/Reporting

## What does this PR do?



Closes #5440

### [Demo](https://www.loom.com/share/b2cd9f83b0ea418a95804c20f90ab090)
<img width="1189" alt="Screenshot 2022-11-09 at 9 13 48 PM" src="https://user-images.githubusercontent.com/1780212/200875215-9b4e71ad-2907-414f-bfbd-c3f7b577501c.png">

**Environment**: Production

## Type of change




## How should this be tested?



- See loom



_Note: You would see 2 unit tests check below. One is passing and the other is failing. This is because I have a fix for unit tests workflow in my PR which is passing and `main` one is failing_
